### ISSUE TITLE
Use tensorflow addons

### ISSUE THREAD

**Issue by ndiamant** [broadinstitute/ml4h#153]
https://www.tensorflow.org/addons/api_docs/python/tfa/optimizers has `Radam`, cyclic learning rate, and model weight averaging, all of which we want. We should use it.
In progress in https://github.com/broadinstitute/ml/pull/170
